app/controllers/reviewables_controller.rb

```diff
@@ -74,7 +74,8 @@ def index
           total_rows_reviewables: total_rows,
           types: meta_types,
           reviewable_types: Reviewable.types,
-          unknown_reviewable_types: Reviewable.unknown_types,
+          unknown_reviewable_types_and_sources:
+            Reviewable.unknown_types_and_sources.map { |k, v| { type: k, source: v } },
```

Comment:
Small nit but I think `Reviewable.unknown_types_and_sources` should return a `hash` instead of the method caller having to massage the return value into something usable.